Public/Src/FrontEnd/Nuget/NugetAnalyzedPackage.cs

```diff
@@ -70,6 +70,9 @@ public sealed class NugetAnalyzedPackage
         /// <nodoc />
         public string Id => PackageOnDisk.Package.Id;
 
+        /// <nodoc />
+        public string NugetName { get; set; }

```

Comment:
Is this property every used?

Reply:
Yes, it is used in GetOriginalNugetPackageName